README.md

```diff
@@ -20,6 +20,16 @@ pip install -r requirements.txt
 
 In the case of problems, try to first run: ```pip install --upgrade pip setuptools```
 
+### CocoAPI
+
+To run evaluations on detections or instance segmentations, compile the cocoAPI:
+
+```
+git submodule update --init --recursive
+cd cocoapi/PythonAPI
+make
+```
```

Comment:
Maybe also another way to install it: `pip install git+https://github.com/MartinSmeyer/cocoapi.git#subdirectory=PythonAPI`

A small suggestion is to have a different version number to distinguish this pycocotools and the official one.
And maybe this version string can be checked to avoid users using the wrong one.

BTW, it seems the pypi version `pycocotools` is newer than the Github version, do you know how they differs? Does the latest pypi version still have the bug you mentioned?

Reply:
It seems the official pypi version `pycocotools` is maintained here: https://github.com/ppwwyyxx/cocoapi. Maybe it is better to add changes based on this?